homeassistant/components/proximity/sensor.py

```diff
@@ -151,8 +166,10 @@ def __init__(
         self.tracked_entity_id = tracked_entity_descriptor.entity_id
 
         self._attr_unique_id = f"{coordinator.config_entry.entry_id}_{tracked_entity_descriptor.identifier}_{description.key}"
-        self._attr_name = f"{self.tracked_entity_id.split('.')[-1]} {description.name}"
         self._attr_device_info = _device_info(coordinator)
+        self._attr_translation_placeholders = {
+            "tracked_device": self.tracked_entity_id.split(".")[-1]
```

Comment:
In the code we refer to these as tracked entities in I believe.

Reply:
```suggestion
            "tracked_entity": self.tracked_entity_id.split(".")[-1]
```

Reply:
yeah ... you're absolutely right